README.md

```diff
@@ -798,8 +798,8 @@ Keep in mind that many predefined parsers and some of the combinators have a var
 | `stringsSepBy1` | `Many1Strings(P<string>, P<String>)` |
 | `skipped` | `P<string> P<Unit>.WithSkipped()` |
 | `withSkippedString` | `P<T2> P<T1>.WithSkipped(Func<string, T1, T2>)`,<br>`P<(string,T)> P<T>.WithSkipped()` |
-| `numberLiteral` | not implemented |
-| `numberLiteralE` | not implemented |
+| `numberLiteral` | `P<NumberLiteral> NumberLiteral<U>(NumberLiteralOptions, string)` |
+| `numberLiteralE` | `P<NumberLiteral> NumberLiteralE<U>(NumberLiteralOptions, ErrorMessageList, CharStream<U>)` |
```

Comment:
Actually returns a `Reply<T>` instead of a `P<T>`.
Also list main variant w/o user state.
```suggestion
| `numberLiteralE` | `Reply<NumberLiteral> NumberLiteralE(NumberLiteralOptions, ErrorMessageList, CharStream<Unit>)` |
```